plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/history/HistoryPresenter.java

```diff
@@ -132,9 +132,9 @@ public void showDialog(Project project) {
         if (isViewClosed) {
             workspaceAgent.openPart(this, PartStackType.TOOLING);
             isViewClosed = false;
-        } else {
-            partStack.setActivePart(this);
         }
+
+        partStack.setActivePart(HistoryPresenter.this);
```

Comment:
Yes, thanks.
This line was in the inner class and I forgot to remove prefix `HistortyPresenter.`